sources/packages/backend/apps/api/src/testHelpers/auth/institution-auth-helpers.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  Institution,
+  InstitutionLocation,
+  InstitutionUser,
+  InstitutionUserAuth,
+  InstitutionUserRoles,
+  InstitutionUserTypeAndRole,
+  InstitutionUserTypes,
+  User,
+} from "@sims/sims-db";
+import { createFakeInstitutionUser } from "@sims/test-utils";
+import {
+  COLLEGE_C_BUSINESS_GUID,
+  COLLEGE_D_BUSINESS_GUID,
+  SIMS2_COLLC_USER,
+  SIMS2_COLLD_USER,
+  SIMS_COLLC_ADMIN_LEGAL_SIGNING_USER,
+  SIMS_COLLD_ADMIN_NON_LEGAL_SIGNING_USER,
+} from "@sims/test-utils/constants";
+import { DataSource, IsNull } from "typeorm";
+import { InstitutionTokenTypes } from "./institution-token-helpers";
+
+/**
+ * Get the institution and user associated with the institution user token type.
+ * @param dataSource allow access to the database.
+ * @param userType type of the institution user.
+ * @returns institution and user associated with the institution user token type.
+ */
+export async function getAuthRelatedEntities(
```

Comment:
As a suggestion the comment can include that it gets the data which is populated by data seed as it will help someone newly looking into API test cases to understand how the institution fake data is created.

Reply:
Added "This data is currently created on DB by the test-db-seeding prior to E2E tests execution."